partner_firstname/static/description/index.html

```diff
@@ -0,0 +1,490 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
On Odoo.sh to have the green light on branch, we need to remove this line

Reply:
This would have to be said in the mainteners repo as it creates it only when you pass the pre-commit.